build_scripts/windows/scripts/build.cmd

```diff
@@ -193,14 +210,21 @@ for /d %%d in ("azure*.dist-info") do (
 )
 popd
 
-if %errorlevel% neq 0 goto ERROR
 
-echo Building MSI...
-msbuild /t:rebuild /p:Configuration=Release /p:Platform=%ARCH% %REPO_ROOT%\build_scripts\windows\azure-cli.wixproj
+if "%TARGET%"=="msi" (
+    echo Building MSI...
+    msbuild /t:rebuild /p:Configuration=Release /p:Platform=%ARCH% %REPO_ROOT%\build_scripts\windows\azure-cli.wixproj
+) else (
+    echo Building ZIP...
+    REM Rename zip root folder to "Azure CLI"
+    ren %BUILDING_DIR% "Azure CLI"
```

Comment:
```
set BUILDING_DIR=%ARTIFACTS_DIR%\cli
set ZIP_DIR=%ARTIFACTS_DIR%\Azure CLI
```

`ren %BUILDING_DIR% "Azure CLI"` renames `BUILDING_DIR` to `%ARTIFACTS_DIR%\Azure CLI`. Let's not hard code `Azure CLI` but use a variable.